requirements/test.in

```diff
@@ -8,4 +8,4 @@ mock                      # mocking library
 pytest-catchlog           # Show log output for test failures
 pytest-cov                # pytest extension for code coverage statistics
 pytest-django             # pytest extension for better Django support
-responses                 # utility library for mocking out the requests library
+responses                 # utility library for mocking out the requests library
```

Comment:
Hehe, maybe the scripts in `make` should check for this (because they deleted them).